engines/aio/libreoffice-generate-registry.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I can see this script duplicated in `engines/libreoffice/libreoffice-generate-registry.sh`.